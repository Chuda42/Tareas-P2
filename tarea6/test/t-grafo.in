# prueba de tiempo grafo
tiempo_grafo 8000
Fin
