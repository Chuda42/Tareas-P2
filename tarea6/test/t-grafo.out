1>#  prueba de tiempo grafo.
2>
3>Fin.
